docs/services/navigation/_index.md

```diff
@@ -554,3 +554,137 @@ obstacles = await my_nav.get_obstacles()
+angularVelocity, err := imu.AngularVelocity{context.Background, nil}
+```
+
+If you want to get an angular velocity reading, first [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+The movement sensor API's [`GetAngularVelocity()`](/components/movement-sensor/#getangularvelocity) takes angular velocity readings.
+Use angular velocity readings to determine the speed and direction at which your robot is rotating.
+
+### Position
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of the [movement sensor](/components/movement-sensor/) component take position measurements:
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+
+An example of a `Position` reading:
+
+``` go
+// position is a geo.Point{Lat: 40.7, Long: -73.98}, altitude is a float64 100
+position, altitude, err:= imu.Position{context.Background, nil}
+```
+
+If you want to get a position, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+The movement sensor API's [`GetPosition()`](/components/movement-sensor/#getposition) takes position readings.
+The suggested components allow you to get the *absolute* position of components through their Positions readings for use in the [motion service](/services/motion/) and [navigation service](/services/navigation/).
+Use position readings to determine the GPS coordinates of an object in 3D space or its position in the geographic coordinate system [(GCS)](https://en.wikipedia.org/wiki/Geographic_coordinate_system).
+
+### Linear Velocity
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/) take linear velocity measurements:
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+- [wheeled-odometry](/components/movement-sensor/wheeled-odometry/) (provides a relative estimate only based on where the base component has started)
+
+An example of a `Linear Velocity` reading:
+
+``` go
+// linearVelocity is an r3.Vector{X:0, Y:1, Z:0}
+linearVelocity, err := imu.LinearVelocity{context.Background, nil}
+```
+
+If you want to get linear velocity, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+The movement sensor API's [`GetLinearVelocity()`](/components/movement-sensor/#getlinearvelocity) takes linear velocity readings.
+Use linear velocity readings to determine the speed at which your robot is moving through space.
+Use [linear acceleration](/services/navigation/#linear-acceleration) readings from another movement sensor to determine the rate of change of this speed.
```

Comment:
```suggestion
Use linear velocity readings to determine the speed at which your robot is moving through space.

To get linear velocity, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
Then use the movement sensor API's [`GetLinearVelocity()`](/components/movement-sensor/#getlinearvelocity) method to get linear velocity readings from the sensor.
```
Same order change suggestion as the rest. Also I think the linear acceleration stuff being here may be more confusing than helpful, and they can look at the linear acceleration section right below if that's what they're after.

Reply:
ok, sounds good. I'm fine with that! Done